cmd/clustertree/cluster-manager/app/manager.go

```diff
@@ -126,9 +124,8 @@ func run(ctx context.Context, opts *options.Options) error {
 		}
 	}()
 
-	masterResourceManager.InformerFactory.Start(ctx.Done())
-	masterResourceManager.KosmosInformerFactory.Start(ctx.Done())
-	if !cache.WaitForCacheSync(ctx.Done(), masterResourceManager.EndpointSliceInformer.HasSynced) {
+	rootResourceManager.InformerFactory.Start(ctx.Done())
+	if !cache.WaitForCacheSync(ctx.Done(), rootResourceManager.EndpointSliceInformer.HasSynced) {
 		klog.Fatal("Knode manager: wait for informer factory failed")
```

Comment:
Why delete this line?
```
masterResourceManager.KosmosInformerFactory.Start(ctx.Done())
```

Reply:
I thought this might be a mistake, so I kept the line and just changed the name from: 
```
	masterResourceManager.InformerFactory.Start(ctx.Done())
	masterResourceManager.KosmosInformerFactory.Start(ctx.Done())
```
to:
```
	rootResourceManager.InformerFactory.Start(ctx.Done())
	rootResourceManager.KosmosInformerFactory.Start(ctx.Done())
```